### ISSUE TITLE
bad bot blocker is useful in conjunction with ModSecurity?

### ISSUE THREAD

**Issue by matteoraggi** [bluedragonz/bad-bot-blocker#14]
My fastcomet hosting is already using ModSecurity firewall. In this case is still ussefull to use this script or not?

**Comment by slrslr** [comment]
I do not think that hosting providers will block mentioned spiders, maybe only a few worst of them. So i think the bad-bot-blocker htaccess rules are still useful. I listen that large .htaccess can slow down website loading, not sure if it apply also in this case.
